src/github/api/mod.rs

```diff
@@ -261,23 +261,10 @@ impl fmt::Display for RepoPermission {
     }
 }
 
-#[derive(serde::Deserialize, Debug)]
-pub(crate) struct OrgAppInstallation {
-    #[serde(rename = "id")]
-    pub(crate) installation_id: u64,
-    pub(crate) app_id: u64,
-}
-
-#[derive(serde::Deserialize, Debug)]
-pub(crate) struct RepoAppInstallation {
-    pub(crate) name: String,
-}
-
 #[derive(serde::Deserialize, Debug, Clone)]
 pub(crate) struct Repo {
     pub(crate) node_id: String,
     #[serde(rename = "id")]
-    pub(crate) repo_id: u64,
```

Comment:
apparently this was only used for github app stuff.

Reply:
Yeah, I remember adding it because of the app sync logic.